.vscode/settings.json

```diff
@@ -5,5 +5,6 @@
       "*.props": "xml",
       "*.targets": "xml",
       "*.tasks": "xml"
-    }
+    },
+    "dotnet.preferCSharpExtension": true
```

Comment:
cSharp extension only supports SDK style projects, this is fine since we can only build SDK style on linux